confgenerator/confgenerator_test.go

```diff
@@ -30,7 +30,10 @@ import (
 const (
 	validTestdataDir   = "testdata/valid"
 	invalidTestdataDir = "testdata/invalid"
-	defaultConfig      = "default-config.yaml"
+	// Relative path to the ops-agent/confgenerator folder.
```

Comment:
The above paths are also relative to the `confgenerator` folder. Worth moving the comment up?

Reply:
Yup. Moved it up.